Index: app/src/main/java/net/ifeu/edicards/Constants/ConstantsTypes.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/net/ifeu/edicards/Constants/ConstantsTypes.java b/app/src/main/java/net/ifeu/edicards/Constants/ConstantsTypes.java
--- a/app/src/main/java/net/ifeu/edicards/Constants/ConstantsTypes.java	
+++ b/app/src/main/java/net/ifeu/edicards/Constants/ConstantsTypes.java	
@@ -20,7 +20,7 @@
 	public static final String NEW_LINE = "\n";
 
 	public static final String NEW_CUSTOMER_CODE = "99";
-	
+	public static final String NEW_CUSTOMER_NTV_CODE = "100000";
 
 	public static final int TIPO_HISTORICO_CLIENTE_EXISTENTE = 1;
 	public static final int TIPO_HISTORICO_CLIENTE_NUEVO = 2;
Index: app/src/main/java/net/ifeu/edicards/DepositManager.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/net/ifeu/edicards/DepositManager.java b/app/src/main/java/net/ifeu/edicards/DepositManager.java
--- a/app/src/main/java/net/ifeu/edicards/DepositManager.java	
+++ b/app/src/main/java/net/ifeu/edicards/DepositManager.java	
@@ -1963,7 +1963,6 @@
 				} catch (Exception e) {
 					throw new RuntimeException(e);
 				}
-				_appConfig.getWorkingArea().CurrentCliente = (Cliente) payload;
 				break;
 			}
 
@@ -1989,45 +1988,75 @@
 			}
 
 			case "EventNtvImportStarted": {
+
 				DepositoNTVDTO depositoNTVDTO = (DepositoNTVDTO) payload;
 
-				_cliente = Factory.build(Cliente.class, _appConfig);
-				boolean exists;
-				try {
-					exists = _cliente.setClienteByCodigo(depositoNTVDTO.IdCliente);
-				} catch (Exception e) {
-					throw new RuntimeException(e);
-				}
+				if (depositoNTVDTO.IdCliente.equals(ConstantsTypes.NEW_CUSTOMER_NTV_CODE)) {
+					try {
+						_cliente = Factory.build(Cliente.class, _appConfig);
+						_cliente.setClienteByCodigo(ConstantsTypes.NEW_CUSTOMER_CODE);
+						_appConfig.getWorkingArea().CurrentCliente = _cliente;
+
+						try {
+							_deposito = Factory.build(Deposito.class, _appConfig);
+							_deposito.assingFromCliente(_cliente);
+							_deposito.IsNtvDeposit = true;
+							this.CreateDepositView(false, depositoNTVDTO);
+						} catch (Exception e) {
+							throw new RuntimeException(e);
+						}
+
+						this.CreateDepositView(false, depositoNTVDTO);
+
+						this._dialogDepositoModalidad = new AdvancedMessageBox();
+						boolean resultDepositoModalidad = _dialogDepositoModalidad.Show("Gestión de Depósito", "Qué tipo de albarán Deseas ?", "Entregar mercancía físicamente", "Enviar desde Edicards", DepositManager.this.getContext(), MessageBoxType.Information);
+						this._appConfig.getWorkingArea().CurrentDepositoModalidad = resultDepositoModalidad ? DepositoModalidad.Furgoneta : DepositoModalidad.Edicards;
+						TextView labelTipoEntrega = (TextView) getActivity().findViewById(R.id.lblTipoEntrega);
+
+						if (labelTipoEntrega != null)
+							labelTipoEntrega.setText(_appConfig.getWorkingArea().CurrentDepositoModalidad == DepositoModalidad.Edicards ? "Enviar desde Edicards" : "Entregar mercancia físicamente");
+
+						} catch (Exception e) {
+							throw new RuntimeException(e);
+						}
+				} else {
+					_cliente = Factory.build(Cliente.class, _appConfig);
+					boolean exists;
+					try {
+						exists = _cliente.setClienteByCodigo(depositoNTVDTO.IdCliente);
+					} catch (Exception e) {
+						throw new RuntimeException(e);
+					}
 
-				if (!exists) {
-					_appConfig.getMessageBox().Show("Atención",
-							"No se han encontrado el cliente especificado " + depositoNTVDTO.IdCliente,
-							_appConfig, MessageBoxType.Error);
-					return;
-				}
+					if (!exists) {
+						_appConfig.getMessageBox().Show("Atención",
+								"No se han encontrado el cliente especificado " + depositoNTVDTO.IdCliente,
+								_appConfig, MessageBoxType.Error);
+						return;
+					}
 
-				_appConfig.getWorkingArea().CurrentCliente = _cliente;
+					_appConfig.getWorkingArea().CurrentCliente = _cliente;
 
-				try {
-					_deposito = Factory.build(Deposito.class, _appConfig);
-					_deposito.assingFromCliente(_cliente);
-					_deposito.IsNtvDeposit = true;
-					this.CreateDepositView(false,depositoNTVDTO);
-				} catch (Exception e) {
-					throw new RuntimeException(e);
-				}
+					try {
+						_deposito = Factory.build(Deposito.class, _appConfig);
+						_deposito.assingFromCliente(_cliente);
+						_deposito.IsNtvDeposit = true;
+						this.CreateDepositView(false, depositoNTVDTO);
+					} catch (Exception e) {
+						throw new RuntimeException(e);
+					}
 
-				this._dialogDepositoModalidad = new AdvancedMessageBox();
-				boolean resultDepositoModalidad = _dialogDepositoModalidad.Show("Gestión de Depósito", "Qué tipo de albarán Deseas ?", "Entregar mercancía físicamente", "Enviar desde Edicards", DepositManager.this.getContext(), MessageBoxType.Information);
-				this._appConfig.getWorkingArea().CurrentDepositoModalidad = resultDepositoModalidad ? DepositoModalidad.Furgoneta : DepositoModalidad.Edicards;
-				TextView labelTipoEntrega = (TextView) getActivity().findViewById(R.id.lblTipoEntrega);
-				labelTipoEntrega.setText(_appConfig.getWorkingArea().CurrentDepositoModalidad == DepositoModalidad.Edicards ? "Enviar desde Edicards" : "Entregar mercancia físicamente");
-				try {
-					this.refreshTotals();
-				} catch (Exception e) {
-					throw new RuntimeException(e);
+					this._dialogDepositoModalidad = new AdvancedMessageBox();
+					boolean resultDepositoModalidad = _dialogDepositoModalidad.Show("Gestión de Depósito", "Qué tipo de albarán Deseas ?", "Entregar mercancía físicamente", "Enviar desde Edicards", DepositManager.this.getContext(), MessageBoxType.Information);
+					this._appConfig.getWorkingArea().CurrentDepositoModalidad = resultDepositoModalidad ? DepositoModalidad.Furgoneta : DepositoModalidad.Edicards;
+					TextView labelTipoEntrega = (TextView) getActivity().findViewById(R.id.lblTipoEntrega);
+					labelTipoEntrega.setText(_appConfig.getWorkingArea().CurrentDepositoModalidad == DepositoModalidad.Edicards ? "Enviar desde Edicards" : "Entregar mercancia físicamente");
+					try {
+						this.refreshTotals();
+					} catch (Exception e) {
+						throw new RuntimeException(e);
+					}
 				}
-
 				break;
 			}
 		}
