Index: app/src/main/java/net/ifeu/edicards/DepositManager.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/net/ifeu/edicards/DepositManager.java b/app/src/main/java/net/ifeu/edicards/DepositManager.java
--- a/app/src/main/java/net/ifeu/edicards/DepositManager.java	
+++ b/app/src/main/java/net/ifeu/edicards/DepositManager.java	
@@ -539,7 +539,8 @@
 
 	private void CreateDepositView() throws Exception {
 
- 		if (this.RestriccionIngresos()) return;
+		//PVT : RAUL
+ 		//if (this.RestriccionIngresos()) return;
 		this.prepareScreenRegions(true);
 
 		Cliente cliente = _appConfig.getWorkingArea().CurrentCliente;
